src/components/LinksListInterface.vue

```diff
@@ -0,0 +1,112 @@
+<template>
+  <div
+    :title="'List for: ' + doi"
+  >
+    <div
+      v-if="(doi)"
+    >
+      <div v-if="typeof dataInput == 'undefined'">
```

Comment:
This applies to all the code, but wouldn't it be better to be explicit about dataInput, make it an empty array or string and only do something with the data if you have it.
That way you don't need to duplicate the render code you have here.